internal/pkg/agent/install/install.go

```diff
@@ -475,8 +475,10 @@ func hasAllSSDs(block ghw.BlockInfo) bool {
 
 func CreateInstallMarker(topPath string, ownership utils.FileOwner) error {
 	markerFilePath := filepath.Join(topPath, paths.MarkerFileName)
-	if _, err := os.Create(markerFilePath); err != nil {
+	handle, err := os.Create(markerFilePath)
```

Comment:
🤦 nice catch. we should have a linter for this

Reply:
So, the windows CI tests will fail on this if you leave the file open in a temp directory created by the golang `testing` library.